tests/integration/devicetemplate/suite_test.go

```diff
@@ -0,0 +1,169 @@
+package devicetemplate
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/cnrancher/edge-api-server/pkg/apis/edgeapi.cattle.io/v1alpha1"
+	"github.com/cnrancher/edge-api-server/tests/integration/cluster"
+	"github.com/cnrancher/edge-api-server/tests/integration/framework"
+	"github.com/stretchr/testify/suite"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/apimachinery/pkg/watch"
+)
+
+type DeviceTemplateTestSuite struct {
+	suite.Suite
+	k8s *framework.K8sCli
+
+	exampleRevisionYAML       []byte
+	exampleDeviceTemplateYAML []byte
+
+	revisionGVR       schema.GroupVersionResource
+	deviceTemplateGVR schema.GroupVersionResource
+
+	deviceTemplate *v1alpha1.DeviceTemplate
+	revision       *v1alpha1.DeviceTemplateRevision
+	cancel         context.CancelFunc
+}
```

Comment:
can we please add a global `TestSuite` and make the `deviceTemplate` as test cases of the suite.